### DISCUSSION TITLE
Issue to combine react-native-firebase and react-native-intercom

### DISCUSSION THREAD

**Discussion by atrofimenko** [invertase/react-native-firebase#4645] [Q&A]
Hello! i have an issue on android, i can't force to work these packages together. I faced with an issue, that notifications only works in background state, can't receive an event of new notification when app is in active state. I can found some topics, that you need to create only one service in androidmanifest for all of them, create a subclass RNPushNotificationListenerService, and wright your logic there, but as i understand, all these topics connected with react-native firebase 5th version, in my current version 7.9.2, these services described in androidmanifest of a package and it uses service and receiver for this and usage of another classes. May be someone already faced with this problem and can share his result?

**Comment by mikehardy** [comment]
Only replying to say current version is v10.1.1, incidentally, but I have no idea how to integrate intercom with FCM in the react-native-firebase post-5.x.x world, sorry
